lib/direct_message_channel_creator.rb

```diff
@@ -1,26 +1,25 @@
 # frozen_string_literal: true
 
 module DiscourseChat::DirectMessageChannelCreator
-  attr_reader :chat_channel, :users
-
-  def self.create!(users)
-    direct_messages_channel = DirectMessageChannel.for_user_ids(users.map(&:id).uniq)
+  def self.create!(acting_user:, target_users:)
```

Comment:
Hmm I don't really understand why we need acting_user in this PR? Don't see it being used anywhere.

Reply:
In general don't think it's an issue to add a new thing in a PR that you aren't explicitly using, as long as you are about to use it very soon (in my case I would use it tomorrow)...but I see your point about the `git blame` which I use all the time, so I will defer this change until I work on ignore/mute etc.